po/pt.po

```diff
@@ -105,21 +101,20 @@ msgstr "GiB"
 
 #: src/Widgets/Search.vala:14
 msgid "Search Process"
-msgstr "Buscar por um processo"
+msgstr "Procurar processo"
 
 #: src/Widgets/Search.vala:15
-#, fuzzy
 msgid "Type process name or PID to search"
-msgstr "Digite o nome de um processo ou um PID"
+msgstr "Escreva o nome do processo ou PID a pesquisar"
 
 #: src/Widgets/Statusbar/Statusbar.vala:12
 #: src/Widgets/Statusbar/Statusbar.vala:16
 msgid "Calculating…"
-msgstr ""
+msgstr "A calcular..."
```

Comment:
```suggestion
msgstr "A calcular…"
```

Use ellipses (…) instead of three periods which the source string uses.

Reply:
I've fixed that.
Done